src/tools/db-upgrade/DbUpgrader.cpp

```diff
@@ -22,10 +24,15 @@ DEFINE_string(dst_db_path,
               "multi paths should be split by comma");
 DEFINE_string(upgrade_meta_server, "127.0.0.1:45500", "Meta servers' address.");
 DEFINE_uint32(write_batch_num, 100, "The size of the batch written to rocksdb");
-DEFINE_uint32(upgrade_version,
+DEFINE_uint32(raw_data_version,
```

Comment:
The name is still very difficult to understand

Reply:
Yeah? I feel more intuitive